float64 hBackup
float64 hSafety
float64 hCurrent
float64 tSafety
float64 tBackup
float64 vBackup
float64 lambdaBackup
float64 lambdaSafety
float64 lamda
float64[4] vDesRaw
float64[4] vDes
float64[4] vBak
float64[4] vAct

